spec/features/user_navigates_index_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+feature 'user goes to root path' do
+
+  scenario 'user sees a list of parks' do
+    park1 = FactoryBot.create(:park)
+    park2 = FactoryBot.create(:park)
+
+    visit parks_path
+
+expect(page).to have_content `#{park1.name}`
+expect(page).to have_content `#{park2.name}`
```

Comment:
Stringifying a string interpolation isn't necessary -- please change `#{park2.name}` to just park2.name :)